src/kahypar.jl

```diff
@@ -148,11 +148,21 @@ end
 
 function recursive_bipartite_optimize(bipartiter, code::EinCode, size_dict)
     ixs, iy = getixsv(code), getiyv(code)
-    ixv = [ixs..., iy]
+    if isempty(iy)
```

Comment:
Please pivot the tree regardless `iy` is empty or not, such that the pivot can be tested extensively. We do not care about the time cost of pivot.

Reply:
If `iy` is empty, no dummy tensor is added and how can we pivot the tree?

Reply:
Always add a dummy tensor?

Reply:
I have fixed this issue in the latest commit.